config/rbac/role.yaml

```diff
@@ -159,13 +155,6 @@ rules:
   - get
   - list
   - watch
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - delete
-  - list
```

Comment:
we might still need it for Rook operator. can you please check

Reply:
Again, these are auto-generated changes. If it would have used by any code then it wouldn't have been removed.

Reply:
Also, does rook operator perform any action on namespaces?